### PR TITLE
CppInterOp ReadMe content added

### PR DESCRIPTION
**Pull request by QuillPusher**
Added simple description for new comers, technical advantages for experienced users.

> Not sure of the timeline of the CppInterOp features ported to Clang-REPL/ LLVM, so they have been referenced as upcoming features (described in future tense)

Request for Review: @vgvassilev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
-# InterOp
-A Clang-based C++ Interoperability library
+## CppInterOp Introduction
+The CppInterOp library (previously LibInterOp) provides a minimalist approach for other languages to identify C++ entities (variables, classes, etc.). This enables interoperability with C++ code, bringing the speed and efficiency of C++ to simpler, more interactive languages like Python.
```
Comment (vgvassilev) [inline README.md:2]:
```suggestion
The CppInterOp library provides a minimalist approach for other languages to identify C++ entities (variables, classes, etc.). This enables interoperability with C++ code, bringing the speed and efficiency of C++ to simpler, more interactive languages like Python.
```

**Review by vgvassilev — COMMENTED** [review]
Could we break the lines at 80 cols?

**Comment by vgvassilev** [comment]
@davidlange6, could you make a pass over this PR?

**Comment by vgvassilev** [comment]
Is this PR ready to review?

File: README.md (hunk 1 above)
Comment (QuillPusher) [reply README.md]: resolved in latest push

**Comment by QuillPusher** [comment]
Yes, Ready for Review, was adding build instructions with @Krishna-13-cyber 's help

**Review by vgvassilev — APPROVED** [review]

**Pull request merged**